docs/install.md

```diff
@@ -559,10 +559,13 @@ Select the following tabs for the installation description of your target.
 
 ## 4. First logon on DietPi
 
+The hostname of the system will be: **dietpi**.  
+You might change the name before the first boot within the configuration file `dietpi.txt`.
+
 After the system has booted up, you can continue following the instructions on the screen, or connect via network:
 
 - If you have a keyboard and a monitor connected to your system you login via this console.
-- If you have a headless system (e.g. an SBC without keyboard resp. monitor) you have to use a terminal program (e.g. `putty`) to connect to the system via an ssh connection.
+- If you have a headless system (e.g. an SBC without keyboard resp. monitor) you have to use a terminal program (e.g. `PuTTY`) to connect to the system via an ssh connection.
```

Comment:
Maybe change terminal to SSH?

Reply:
I agree, the sentence should be enhanced, and we could make clear that SSH and even UART as 3rd login possibility is enabled by default on DietPi.
```
- If you have a headless system with no keyboard and monitor attached, you can use an SSH client like [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) to connect from a remote system. The SSH server [Dropbear](../software/ssh/#dropbear) is installed and enabled by default on DietPi.
- Most SBCs alternatively allow to connect a serial console via UART, which is by default enabled on DietPi.
```

Reply:
Thx for the correction. In other places we use "terminal" for a shell window.
Changed from "...you have to use a terminal program..." to "...you have to use an SSH client...". This corresponds to other places in this doc.

Reply:
I added the info about UART, as this is the only way to connect to a headless system when either intentionally or due to an error or false network setup, network is not brought up on first boot. Good to have all three methods listed. I hope you're fine with the wording and some indentation and warning box changes, I couldn't hold myself when scrolling through the file 😅.